README.md

```diff
@@ -86,6 +89,40 @@ You shouldn't be using a root user. Below is a snippet of an inline policy with
 }
 ```
 
+ * This restrict the action to uploading an object and listing/getting the object so it can obtain the location for CodeDeploy
+ * It is restricted to a specific bucket.
+
+For deploying via CodeDeploy you will need another set of permissions.
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "codedeploy:CreateDeployment"
+            ],
+            "Effect": "Allow",
+            "Resource": [
+                "arn:aws:codedeploy:codedeploy-arn"
+            ]
+        },
+        {
+            "Action": [
+                "codedeploy:Batch*",
+                "codedeploy:Get*",
+                "codedeploy:List*",
+                "codedeploy:RegisterApplicationRevision"
+            ],
+            "Effect": "Allow",
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+ * These permissions are a rough example of allowing the user to list/get/register a revision for all
+ * A specific permission statement exists to lock creating the deployment to a specific resource
```

Comment:
Is this saying that there is something I can Google and add to lock creating the deployment or that this snippet will do this. The `Resource` key seems to do this, so I suspect the latter. I just want to understand fully.

Reply:
It just explains the statement in the permission that we treat the creating the deployment on a specific resource.